sigopt/validate/aiexperiment_input.py

```diff
@@ -29,17 +29,28 @@ def get_validated_metrics(experiment_input):
   validated_metrics = []
   for metric in metrics:
     validated_metric = {}
-    if not is_mapping(metric):
-      raise ValidationError("all metrics must be a mapping of keys to values")
-    metric = dict(metric)
+
+    if isinstance(metric, str):
+      metric = {"name": metric}
```

Comment:
this seems like an additional feature? so long as it's happening clientside rather than serverside, I think it's fine.

Reply:
It is supported serverside, which is which i decided to fix the validation client side (even if it does convert the format). This kinda makes the validation easier, but I could also refactor single metric validation which would make it easy to keep the string shorthand.

Reply:
Yeah, I'd like to move away from the multiple allowable formats server-side; we haven't changed them before to avoid backwards-incompatible changes but if these are going to new endpoints, we can enforce metric mappings without breaking workflows (since they'll already need to update anyway, I think?)